test/widgets/test_mpd2widget.py

```diff
@@ -18,6 +18,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import sys
+from collections import defaultdict
```

Comment:
Why is this needed? It doesn't look like your test calls this.

Reply:
> Why is this needed? It doesn't look like your test calls this.

Oh, my bad! I was testing some stuff and forgot to remove this.